scripts/zones/Chateau_dOraguille/npcs/Curilla.lua

```diff
@@ -15,6 +15,7 @@ require("scripts/globals/utils")
 -----------------------------------
 
 local wsQuest = tpz.wsquest.savage_blade
+local questList = tpz.quest.id.sandoria
```

Comment:
Caching this table here, and then bailing out if you're not Sandorian at the start of the trigger logic has the unpleasant side effect of making it hard to parse what's going on if you dive into the middle of the file.
In order to know "Oh this is for Sandorians only" or "all of these missions are for Sandoria" you have to scroll up and read the whole file.
I don't think any of the tabled quest id's are so verbose that they benefit from this improvement.

Reply:
@zach2good I set this as a top priority variable to re-use it in `onEventFinish()` also. It's probably the most used array in the whole script. I was thinking it makes sense to cache it right away for that reason.
Alternatively it could be cached once in the missions branch of `onTrigger()` and once in `onEventFinish()`. Or well, just revert it all back to individually call it every single time throughout the script. 
What do you think?

Reply:
Also how would you prefer to organize the incredible amount of parallelism in such a case? I branched the San d'Oria missions into their own condition so they can be skipped fast if you're just RDM or somebody doing Wildcat etc. I think this behavior is also very well documented in the file and does not require scanning a lot. The variable name is quite verbose as well.

Curilla was looking like Halver, you had to read every line very carefully to try to get a grasp of what's going on. I think opening a path for missions on any NPC might be a good idea to reduce requests. Why would a Bastok MNK need to go through all those mission checks?
Would like to hear your opinion for feedback here :)

Reply:
I think my issue here is that `questList` doesn't give away that it's Sandorian quests without further context.  But if you're already looking in that file, you probably know the context, so I'm probably worrying over nothing.

I wasn't saying that a Bastoker would need to go through those checks, I was suggesting that the reader would be doing a CTRL+SHIFT+F or some other operation that dumps them in the middle of the file and then they have to spend more time figuring out whats going on.